server/index.js

```diff
@@ -33,16 +33,29 @@ const hbs = exphbs.create({
     formatDate(date) {
       return date.toLocaleDateString();
     },
+    isContact(user, currentUser) {
+      return user.contacts.find((contact) => {
+        return contact.id === currentUser.id;
+      });
+    },
+    hasRequest(user, currentUser) {
+      return user.requests.find((req) => {
+        return req.id === currentUser.id;
+      });
+    },
+    isEqual(o1, o2) {
+      return o1 === o2;
+    },
     isLiked(likes, user) {
-      for(var i = 0; i < likes.length; i++) {
-        if(likes[i].id === user.id) {
+      for (let i = 0; i < likes.length; i++) {
```

Comment:
U JavaScriptu se skoro ikada ne koriste for petlje, nezz zasto ali vjerojatno jer ljepše izgleda ovako:
``` javascript
return !!likes.filter((like) => like.id === user.id).length
```
ovo će ustvari filtrirati samo likeove koji imaju isti id kao user, pogledat će duljinu nastalog niza i vratiti bool vrijednost (! radi negaciju, !! radi negaciju negacije, odnosno vraća bool vrijednost izraza) (0 je false u js-u)